homework.js

```diff
@@ -42,6 +75,27 @@
 // https://courses.cs.vt.edu/csonline/Algorithms/Lessons/SelectionSort/index.html
 // to see how. This may make more sense to you.
 
+function selectionSort(x) { 
+    
+    let n = x.length;
+    let x2 = x.slice(0);    
+    for(let i = 0; i < n; i++) {
+        let min = i;
+        for(let j = 0; j < n; j++){
+            if(x2[j] < x2[min]) {
+                min=j; 
+            }
+         }
+         if (min != i) {
```

Comment:
What was your intention with these lines of code? Maybe writing that out will help it be clearer. I'm not sure why you need the loop inside the loop.